docs/nos-client/api.md

```diff
@@ -5,17 +5,58 @@ sidebar_label: API Overview
 ---
 
 **NOTE: The API is in active development and may change frequently.  Be sure to check back
-regularly. All API functions will have a maximum of ONE argument: a JSON object containing all the necessary info.**
+regularly. All API functions will have a maximum of ONE argument: a JSON object containing all the
+necessary info.**
 
 The nOS client provides a prebuilt API that is still growing.  For any dApp running through the
-client, the API can be accessed via `window.NOS.V1`.  All functions return a
-[`Promise`](https://www.google.com/search?q=js+promise&oq=js+promise&aqs=chrome..69i57j69i60l3.731j0j7&sourceid=chrome&ie=UTF-8), which can be used to determine if the call succeeded or failed.  Some calls will fail if the
-user rejects the action.
+client, the API can be accessed via `window.NOS.V1`.
 
 
+## Events
```

Comment:
As `events` and `Functions` are "big" headers, it would make more sense to give it this format: 

```
Events
---

Continue here...
```

These 3 dashes make it a H1 header with a HR below it

Same for `Functions`

Reply:
@DalderupMaurice Do you mind making this change?  I'm not sure I'm following.  I think you have a better idea.  I've addressed the other things you mentioned.